File: cpp/prime.cpp
#include<iostream>
using namespace std;

int main(){
int n;
cout<<"Enter number";
cin>>n;
int i;
for(i=2;i<n;i++){
	if(n%i==0){
		cout<<n<<" is not prime number"<<endl;
		break;	
	}	
}
if(i==n){
cout<<n<<" is prime number"<<endl;
}
}

File: 0_Assignment/1/prime.java
import java.util.*;
class prime{
	public static void main(String a[]){
		Scanner sc = new Scanner(System.in);	
		System.out.println("Enter Number :");
		int num = sc.nextInt();
		boolean flag = true;
		for(int i=2;i<(num/2);i++){
			if(num%i==0){
				flag = false;
				break;
			}		
		}
		if(flag)
			System.out.println(num+" is prime number");
		else
			System.out.println(num+" is NOT prime number");
	}
}
